packages/cli/src/commands/stream/create.ts

```diff
@@ -0,0 +1,99 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+import { parseControllers } from '../../utils'
+import { StreamMetadata } from '@ceramicnetwork/common'
+
+type Flags = CommandFlags & {
+  'only-genesis'?: boolean
+  controller?: string
+  metadata?: StreamMetadata
+}
+
+export default class Create extends Command<
+  Flags,
+  {
+    content: string
+    schemaId?: string
+  }
+> {
+  static description = 'Create a new Stream'
+
+  static args = [
+    {
+      name: 'schema',
+      description: 'StreamID of desired Schema',
+      required: false,
+    },
+    {
+      name: 'content',
+      description: 'the Stream body',
+      required: true,
+      parse: JSON.parse,
+    },
+  ]
+
+  static flags = {
+    ...Command.flags,
+    'only-genesis': flags.boolean({
+      char: 'g',
+      description: 'only generate genesis block',
+      default: false,
+    }),
+    controller: flags.string({
+      char: 'c',
+      description:
+        'Stream Controller, once set this is the only DID that will be able to update the stream.',
+    }),
+    metadata: flags.string({
+      char: 'm',
+      description: 'Stream Metadata',
+      parse: JSON.parse,
+    }),
+  }
+
+  async run(): Promise<void> {
+    this.spinner.start('Creating stream...')
+    let did: string | undefined
+    if (this.flags.key != null) {
+      did = this.authenticatedDID.id
+    } else {
+      throw new Error('No DID cached, please provide your key.')
+    }
+    try {
+      let parsedControllers: Array<string>
+      if (this.flags.controller !== undefined) {
```

Comment:
I don't think that can work, how can the CLI create a stream with a controller other than the one provided from the key?

Reply:
The reason I did this is purely because (from my understanding) a Tile can be created by one user can have a different user as a controller. It's a decently far edge case and I wasn't convinced by it's use anyways so we'll just remove it & force it to be based off of the user's provided DID.